ansible/roles/grafana/tasks/post_config.yml

```diff
@@ -15,7 +15,7 @@
 
 - name: Remove old grafana docker volume
   become: true
-  kolla_container:
```

Comment:
Not sure about this change

Reply:
Yes I checked the history and I can see that we can use kolla_container again.